src/dlgColorTrigger.cpp

```diff
@@ -404,16 +387,15 @@ void dlgColorTrigger::setColorLightMagenta()
 
 void dlgColorTrigger::setColorWhite()
```

Comment:
:thinking: IIRC if we could set up a `QSignalMappper` we could considerably cut down on the "copy and modify" blocks of largely repetitive code here.